dotguile

```diff
@@ -26,7 +26,7 @@
 ;;; $Id: dotguile,v 1.1 2012/08/26 21:06:27 tuexen Exp $
 
 ;;; Change the following line to reflect where the files are located.
-(define dir "/Users/tuexen/Documents/m3ua-testtool/")
+(define dir "")
```

Comment:
I would prefer to keep the directory name. On the one hand side it shows what to insert, on the other handside it breaks very early if you haven't changed it to an appropriate way.

Reply:
I find it somewhat strange and uncommon that the personal home directory of the developer is hard-coded in some files that are distributed with a project. That's why I was suggesting to move to a "all relative paths' setup where no installation is required, no files to the home directory need to be deployed, etc.

If the home directory must be present in this file, then there should be some template / pattern substitution mechanism that uses the actual "$HOME" at the time of installation.